hack/e2e-kind-cluster-setup.sh

```diff
@@ -1,8 +1,12 @@
 #!/bin/bash
 set -xe
 
+KUBEVIRT_PROVIDER=${KUBEVIRT_PROVIDER:-kind}
```

Comment:
Please explain to me why this project should care about kubevirt. 

It *must* be agnostic to the user.

Reply:
i can change the name of this variable to just say if it is external provider or kind default provider
i just used the name we are using on all repos to be aligned and easier to remember

Reply:
all repos => kubevirt related repos. There it makes sense to have this separation.

This project has *nothing* to do with kubevirt. At most, it is just another user.